draft-ietf-quic-transport.md

```diff
@@ -2227,6 +2243,9 @@ enable an attack on migration; see {{off-path-forward}}.
 An endpoint MUST expand datagrams that contain a PATH_RESPONSE frame to at
 least the smallest allowed maximum datagram size of 1200 bytes. This verifies
 that the path is able to carry datagrams of this size in both directions.
+However, an endpoint MUST NOT expand the PATH_RESPONSE if it is constrained
+by an anti-amplification limit.  This will only occur if the PATH_CHALLENGE
+was not sent in an expanded packet.
```

Comment:
```suggestion
However, an endpoint MUST NOT expand the datagram containing the
PATH_RESPONSE if it is constrained by an anti-amplification limit.  This
will only occur if the PATH_CHALLENGE was not sent in an expanded
datagram.
```
The first sentence says you expand the datagram, so let's not confuse the issue by referring to expanded frames or packets.